app/controllers/concerns/recorder.rb

```diff
@@ -59,7 +98,8 @@ def filter_recordings(api_res, search)
                ((r[:metadata].nil? || r[:metadata][:name].nil?) &&
                  r[:name].downcase.include?(search)) ||
                r[:participants].include?(search) ||
-               !r[:playbacks].select { |p| p[:type].downcase.include?(search) }.empty?
+               !r[:playbacks].select { |p| p[:type].downcase.include?(search) }.empty? ||
+               Room.find_by(bbb_id: r[:meetingID]).owner.email.downcase.include?(search)
```

Comment:
Will this mean that if you search for recordings under a room then it will select recordings based on the owner email even though that isn't displayed in the UI?

Reply:
Fixed